windows/mdns_impl.c

```diff
@@ -0,0 +1,525 @@
+
+#include <stdio.h>
```

Comment:
The library is still included in the PR :(

Reply:
the way the library is set up is that the mdns.h file has all the implementation. The mdns.c files is an example command line app demonstrating how to use it. It includes also code for MDNS server and other aspects. The mdns_impl.c was generated by selective copy/paste and modification of the sample code to implement the specific use case. So this file needs to remain.

Reply:
Ahh ok. That's clear now 👍